pkg/interface/src/views/apps/links/components/lib/link-submit.tsx

```diff
@@ -83,11 +83,12 @@ export class LinkSubmit extends Component<LinkSubmitProps, LinkSubmitState> {
         fetch(`https://noembed.com/embed?url=${linkValue}`)
         .then(response => response.json())
         .then((result) => {
-          if (result.title) {
+          if (result.title && !this.state.linkTitle) {
```

Comment:
possibly better to just not fetch at all if we have a link title?

Reply:
Also really in the case of this — this isn't chat, so it shouldnt be in the chat commit.